jts-test-runner/src/input.rs

```diff
@@ -215,6 +215,11 @@ pub(crate) enum Operation {
         op: BoolOp,
         expected: Geometry<f64>,
     },
+    ClipOp {
```

Comment:
JTS doesn't have a special "clip" operation. Instead it supports boolean operations on LineString x Polygons, so I've mapped those to our BooleanOp::clip method to get some more test coverage.